DuckDuckGo/Base.lproj/Main.storyboard

```diff
@@ -293,12 +293,21 @@
                                 <rect key="frame" x="0.0" y="0.0" width="375" height="2"/>
                                 <color key="progressTintColor" red="0.0" green="0.50196081400000003" blue="1" alpha="1" colorSpace="calibratedRGB"/>
                             </progressView>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Error Message" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="5" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Sel-Fj-CU4">
+                                <rect key="frame" x="61" y="72" width="253" height="23.5"/>
+                                <fontDescription key="fontDescription" name="ProximaNova-Semibold" family="Proxima Nova" pointSize="20"/>
```

Comment:
The design has the text as 16. I think 20 looks good on larger devices so maybe make 16 the default and 20 the size for the the regular-regular size class?

The line height is also 24pt in the design (which is a lineHeightMultiple of 1.5). You could use an attributedString to fix this but since these don't translate too well you can do it in code. There's a UILabel extension method in the project so `errorMessage.adjustPlainTextLineHeight(1.5)` might do what you need, feel free to expand it if it doesn't. Beware the `height` variable is badly named, it takes a multiple which isn't very clear from the name.